cmd/vochaintest/vochaintest.go

```diff
@@ -142,9 +164,9 @@ func main() {
 			log.Fatal(err)
 		}
 		cspVoteTest(*host,
-			*oraclePrivKey,
+			oracleKey,
```

Comment:
Is oracleKey needed? I don't think so since we are using fresh accounts

Reply:
I used for setting the process status to end so is checked that the oracle can set arbitrary processes, but I'm doing it in other functions so we can remove the oracle key here.